ocp_resources/constants.py

```diff
@@ -5,3 +5,4 @@
 PROTOCOL_ERROR_EXCEPTION_DICT = {ProtocolError: []}
 NOT_FOUND_ERROR_EXCEPTION_DICT = {NotFoundError: []}
 TIMEOUT_4MINUTES = 240
+TIMEOUT_8MINUTES = 480
```

Comment:
Used only in one place, should be used directly

Reply:
ACK